cli-feed-v4.json

```diff
@@ -46,7 +46,7 @@
       "hidden": false
     },
     "v4-prerelease": {
-      "release": "4.65.0",
+      "release": "4.66.0",
```

Comment:
are we only updating prerelease one? That means, we need to wait again (Do we know how many days it will be?) to get this changes out to customers. can we update "v4" as well?

Reply:
Not sure about the requirements of how many days. Following guidance from this [doc](https://eng.ms/docs/cloud-ai-platform/devdiv/serverless-paas-balam/serverless-paas-benbyrd/app-service-web-apps/app-service-team-documents/functionteamdocs/engineeringprocesses/releaseprocesses/templatesrelease#part-two-official-nuget-build-and-release) which says - 

_Inside cli-feed-v4.json, bump the minor or patch version (depending on your use case) for the prerelease tag of the Functions version you want to update. E.g. for v4 you would update the tag named v4-prerelease_

@soninaren @liliankasem any thoughts on this?

Reply:
Bumping v4 without properly validation from the .net team is dangerous - we've done that before and it lead to a huge bug with a lot of noise. I would like to avoid that so we should stick with guidance and do the prerelease. You can reach out to Phil to help expedite validation

Reply:
As someone who used to be on the tooling team, I can confirm skipping the prerelease step has caused entirely avoidable pain in the past 😅